src/features/page_objects/check-and-submit.page.js

```diff
@@ -5,60 +5,85 @@ class CheckAndSubmit extends Page {
     return "Check your answers before submitting your information";
   }
 
-  get addDetails() {
-    return $("#add-your-details");
+  get changeFullname(){
+    return $("a[href='/land/name']");
   }
 
-  get addLandBoundary() {
-    return $("#add-your-landboundary");
+  get fullnameValue(){
+    return $("/html/body/div[1]/main/form/div/div/div/dl[1]/div[1]/dd[1]");
   }
 
-  get addLandOwnership() {
-    return $("#add-land-ownership");
+  get roleValue(){
+    return $("/html/body/div[1]/main/form/div/div/div/dl[1]/div[2]/dd[1]");
   }
 
-  get addMetric() {
-    return $("#add-habitat-information");
+  get emailAddressValue(){
+    return $("/html/body/div[1]/main/form/div/div/div/dl[1]/div[3]/dd[1]");
   }
 
-  get addHmmp(){
-    return $("#add-habitat-management");
+
+  get landBoundaryTypeValue(){
```

Comment:
You have good selectors now [data-testid='land-boundary-file-type-value']

Reply:
updated the datatestids that are present now.
new ones to be added in upcoming ticket https://eaflood.atlassian.net/browse/BNGP-1959